app/controllers/PreferencesProxy.scala

```diff
@@ -0,0 +1,47 @@
+package controllers
+
+import com.gu.pandomainauth.PanDomainAuthSettingsRefresher
+import com.gu.workflow.util.{Code, Dev}
+import config.Config
+import play.api.Logging
+import play.api.libs.ws.WSClient
+import play.api.mvc.{BaseController, ControllerComponents}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+/* Proxy for editorial-preferences, to help with latency since workflow-frontend sits behind CloudFront*/
+class PreferencesProxy(
+  override val config: Config,
+  override val controllerComponents: ControllerComponents,
+  override val wsClient: WSClient,
+  override val panDomainSettings: PanDomainAuthSettingsRefresher
+) extends BaseController with PanDomainAuthActions with Logging {
+
+  private def proxyRequest(relativePath: String) = APIAuthAction.async { request =>
+    val url = s"${config.preferencesUrl}/$relativePath"
+    val headers = request.headers.toSimpleMap + (
+      "Host" -> config.preferencesHost,
+      "Cache-Control" -> "private, no-cache, no-store, must-revalidate, max-age=0", // do not cache whatsoever
```

Comment:
This cache-control header should be in the response on line 30, not in the request that you are sending to the preferences service.

Reply:
🤦  - thanks for catching that - updated in https://github.com/guardian/workflow-frontend/pull/408/commits/9641702b301bc220da763a3dc3d791b9836fa5fc - plus better variable name(s)